examples/imagenet/configs/default.py

```diff
@@ -11,20 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-# Copyright 2020 The Flax Authors.
```

Comment:
Was this removed accidentally?

Reply:
The `# Copyright 2021` is enough, no?

Reply:
(I just noted this passing by and thought I would include it in the same CL.)

Reply:
Oh I missed the duplicate. You are right